### PR TITLE
www apps/gitea: bump to 1.8.1

### PR DESCRIPTION
**Pull request by xdch47**
See: https://bugs.gentoo.org/684196

### CODE REVIEW

File: www-apps/gitea/files/gitea.service-r2 [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Description=Gitea service
+Documentation=https://docs.gitea.io/
+
+AssertPathIsDirectory=/var/lib/gitea
+AssertPathIsReadWrite=/var/lib/gitea
+
+After=network.target
+Requires=network.target
+After=mysqld.service
+After=postgresql.service
```
Comment (hydrapolic) [inline www-apps/gitea/files/gitea.service-r2]: Please see https://bugs.gentoo.org/671596. Seems like the postgresql versions need to be included here.

**Comment by xdch47** [comment]
Thanks, it seem there is no wildcard support in systemd yet,
so I listed all, as you suggested. However, it feels like is a not really elegant solution.

- Another solution would be that `eselect-postgresql` should provide a default service file (postgresql.service) by putting a symlink to postgresql-SLOT.service - or am I wrong?

**Comment by hydrapolic** [comment]
> Thanks, it seem there is no wildcard support in systemd yet,
> so I listed all, as you suggested. However, it feels like is a not really elegant solution.
> 
>     * Another solution would be that `eselect-postgresql` should provide a default service file (postgresql.service) by putting a symlink to postgresql-SLOT.service - or am I wrong?

Seems like this is the best we can do right now. 

Yes, that idea with the symlink might work, @titanofold what do you think?

**Comment by xdch47** [comment]
1.8.1 has been released in the meantime.
@hydrapolic - could you create a new PR? Then it'll get merge eventually ;)

**Comment by hydrapolic** [comment]
> @hydrapolic - could you create a new PR? Then it'll get merge eventually ;)

I hope this will get merged :)

**Pull request closed without merging**